.github/actionlint.yaml

```diff
@@ -0,0 +1,10 @@
+self-hosted-runner:
+  # Labels of self-hosted runner in array of strings.
+  labels:
+    - ubuntu-amd64*
+    - ubuntu-x64*
+    - ubuntu-arm64*
+# Configuration variables in array of strings defined in your repository or
+# organization. `null` means disabling configuration variables check.
+# Empty array means no configuration variable is allowed.
+config-variables: null
```

Comment:
why is this one being disabled explicitly? according to the docs, this is the default

Reply:
not sure - this is automatically generated and I left it like this until we actually need it 😄 
are there any config variables you'd like to check now?

Reply:
nope, I think this could be dropped as it's setting a setting to its default value :upside_down_face:

Reply:
removed!